src/index.ts

```diff
@@ -63,6 +70,7 @@ export default class BlueButton {
     this.callbackUrl = bbJsonConfig.callbackUrl;
     this.clientSecret = bbJsonConfig.clientSecret;
     this.version = bbJsonConfig.version;
+    this.callbackUrl = bbJsonConfig.callbackUrl;
```

Comment:
This already exists above ^

Reply:
I think I missed this when resolving merge conflicts. Removed this duplicate. TY!